colour/models/rgb/cylindrical.py

```diff
@@ -31,6 +33,11 @@
     Interactive Techniques - SIGGRAPH "78, 12-19. doi:10.1145/800248.807361
 -   :cite:`Wikipedia2003` : Wikipedia. (2003). HSL and HSV. Retrieved
     September 10, 2014, from http://en.wikipedia.org/wiki/HSL_and_HSV
+-   :cite:`Sarifuddin2005ANP` : M. SarifuddinA (2005) New Perceptually Uniform
```

Comment:
Here are the citation key and bibliography:

- `Sarifuddin2005`
- `Sarifuddin, M., & Missaoui, R. (2005). A New Perceptually Uniform Color Space with Associated Color Similarity Measure for ContentBased Image and Video Retrieval.`